src/quantuminspire/qiskit/qi_job.py

```diff
@@ -16,46 +18,46 @@ class QIJob(BaseJob):
             result = job.result()
     """
 
-    def __init__(self, backend, job_id, api, qobj=None):
+    def __init__(self, backend: Any, job_id: str, api: QuantumInspireAPI, qobj: Optional[Qobj] = None) -> None:
         """
         Construct a new QIJob object. Not normally called directly, use a backend object to create/retrieve jobs.
 
         Args:
             backend (QuantumInspireBackend): A quantum-inspire backend.
```

Comment:
typehint in docstring

Reply:
I left the docstring types for those types that caused recursive import problems [DEM-674], because Any is not very informative, but I deleted this now.